makefile

```diff
@@ -101,7 +101,17 @@ e2e-test: prepare
 	minikube image load $(prevDockerImage)
 
 	@echo "=====Running e2e tests"
-	$(if $(saveOutput),gotestsum --junitfile test/test_results/e2e-tests.xml ./test/e2e/... -count=1 -timeout 70m, go test -v -count=1 -timeout 70m ./test/e2e/...) 
+	$(if $(saveOutput),gotestsum --junitfile test/test_results/e2e-tests.xml ./test/e2e/... -count=1 -timeout 70m, go test -v -count=1 -timeout 70m ./test/e2e/...)
+
+	@echo "=====Apply daemonset to configure huge pages on minikube cluster"
+	kubectl apply -f ./test/test_data/hugepages/daemonset.yaml
+
+	@echo "=====Restart minikube to apply huge pages setting"
+	minikube stop
+	minikube start --driver=docker --kubernetes-version=$(kubernetesVersion) -n=1 --memory=$(minikubeMemory) --cpus=2
```

Comment:
You don't have to specify parameters when you're starting existing minkube. It'll use the same parameters by default.

Reply:
removed the restarting minikube